### PR TITLE
Tenant Modal

### PR DESCRIPTION
**Pull request by adowski**
## New Things

- Styled `TenantModal` component
- Restyled buttons for accepting/rejecting/interviewing
- File download table

### Related PRs

None

### Migrations

None

### Tests Performed, Edge Cases

None

### //TODO

- See if the download buttons work and update the table as soon as there is a form attached to a tenant
- ~~There are warnings from some fileName property thing I will fix~~

copying all of my todos from last PR for my own sake:
- I really want to get rid of the warning that comes from ListView so I will do that soon lol
- ~~also as u can see below, it appears the location column is too small haha darn so will fix that~~
- also ListView is taking in so many props I am confused I will clean that up aioefoajdiao

### Screenshots

image

CC: @wilsonw926

### CODE REVIEW

File: app/javascript/components/PropertyShowPage.js [hunk 1]
```diff
@@ -27,9 +27,19 @@ class PropertyShowPage extends React.Component {
     Utils.setup(this.state.tenantApps, this.props.tenantAppsPDF);
     Utils.setup(this.state.potentialTenantApps, this.props.potentialTenantAppsPDF);
     Utils.setup(this.state.potentialTenants, this.props.potentialTenantStatuses);
+
+    console.log("CURRENT TENANTS")
```
Comment (wilsonw926) [inline app/javascript/components/PropertyShowPage.js]: remove these

File: app/javascript/components/modals/TenantModal.jsx [hunk 2]
```diff
@@ -13,49 +15,127 @@ class TenantModal extends React.Component {
...
+      return(<p><a href={this.props.application.url}><Icon type="paper-clip" />Attachment</a></p>)
     }
   }
 
-  renderAvatar(images){
-    if (typeof(images) === "string") {
-        return (<Avatar size={200} shape="square" src={images} />);
-    } else {
-      return (<Avatar size={200} shape="square" src={images[0].url} />)
+  // renderAvatar(images){
```
Comment (wilsonw926) [inline app/javascript/components/modals/TenantModal.jsx]: remove comments of code

File: app/javascript/components/individual/SplitViewContainer.jsx [hunk 3]
```diff
@@ -22,19 +22,19 @@ const SplitViewContainer = props => (
...
-        display: "inline-block",
-        overflow: "scroll",
-        paddingLeft: "10px",
-        borderLeft: "1px solid gray"
-      }}
-    >
-      {props.rightComponent}
-    </div>
-  </div>
+		<div key="right"
```
Comment (wilsonw926) [inline app/javascript/components/individual/SplitViewContainer.jsx]: fix spacing here to 2 spaces

Comment (wilsonw926) [reply app/javascript/components/individual/SplitViewContainer.jsx]: Also pull from master, i think there may be small merge conflicts here.

File: app/javascript/components/PropertyShowPage.js (hunk 1 above)
Comment (adowski) [reply app/javascript/components/PropertyShowPage.js]: oops done

File: app/javascript/components/modals/TenantModal.jsx (hunk 2 above)
Comment (adowski) [reply app/javascript/components/modals/TenantModal.jsx]: done

File: app/javascript/components/individual/SplitViewContainer.jsx (hunk 3 above)
Comment (adowski) [reply app/javascript/components/individual/SplitViewContainer.jsx]: done @ both

File: app/javascript/components/modals/TenantModal.jsx [hunk 4]
```diff
@@ -13,49 +15,127 @@ class TenantModal extends React.Component {
...
   handleButtons() {
-    if (this.props.housed != null) {
-      if (this.props.housed) {
-        return(<ApplicationStatusButtons property_id={this.props.property_id} application_id={this.props.app.id} status={this.props.app.status}/>)
-      } else {
-        return(<ApplicationStatusButtons property_id={this.props.property_id} application_id={this.props.app.id} status={this.props.app.status}/>)
-      }
-    } else {
-      return(null)
+    if (this.props.housed == null) {
```
[the remark this answers is not in the document]
Comment (adowski) [reply app/javascript/components/modals/TenantModal.jsx]: done
